.circleci/config.yml

```diff
@@ -44,10 +44,25 @@ executors:
 
   deployer:
     docker:
-      - image: omisegoimages/elixir-omg-deploy:stable-20200410
+      - image: omisegoimages/elixir-omg-deploy:dev-da1ecc8
     working_directory: ~/src
 
 commands:
+
+  add_rust_to_path:
+    description: "Add path to PATH env var"
+    steps:
+      - run:
+          name: Add rust to PATH env
+          command: echo 'export PATH=~/.cargo/bin/:$PATH' >> $BASH_ENV
+  install_rust:
```

Comment:
can we specify rust v 1.44 as you did in https://github.com/omisego-images/docker-elixir-omg/pull/45/files#diff-5097196aa827816e990d13cafdea11a1ff82823cee1699a6d74a006d84765eefR66 ?

Reply:
I don't know how to do that with rustup, do you know?

Reply:
`rustup default ...` should work

Reply:
not sure this is gonna work:
```
inomurko@DESKTOP-63PDNVC:/opt/omg-childchain-v1$ rustup default 1.46.0
info: syncing channel updates for '1.46.0-x86_64-unknown-linux-gnu'
info: latest update on 2020-08-27, rust version 1.46.0 (04488afe3 2020-08-24)
error: some components unavailable for download for channel 1.46.0: 'miri', 'rust-analyzer-preview'
If you require these components, please install and use the latest successful build version,
which you can find at <https://rust-lang.github.io/rustup-components-history>.

After determining the correct date, install it with a command such as:

    rustup toolchain install nightly-2018-12-27

Then you can use the toolchain with commands such as:

    cargo +nightly-2018-12-27 build
inomurko@DESKTOP-63PDNVC:/opt/omg-childchain-v1$ echo $?
1
```